packages/clerk-js/src/ui/components/OrganizationProfile/MemberListTable.tsx

```diff
@@ -0,0 +1,141 @@
+import { UserResource } from '@clerk/types';
+import React from 'react';
+
+import { Box, Button, Col, Icon, Table, Tbody, Td, Th, Thead, Tr } from '../../customizables';
+import {
+  InputWithIcon,
+  Menu,
+  MenuItem,
+  MenuList,
+  MenuTrigger,
+  Pagination,
+  usePagination,
+  UserPreview,
+} from '../../elements';
+import { useSearchInput } from '../../hooks';
+import { MagnifyingGlass, ThreeDots } from '../../icons';
+
+const searchTermForUser = (user: any) => {
```

Comment:
❓ Why do we use `any` here?

Reply:
Hey George, thanks for taking a look. This is a placeholder, as the whole component is still a work in progress. This will be replaced with the actual user type once the hooks are in place.